### DISCUSSION TITLE
fingersTuner operation

### DISCUSSION THREAD

**Discussion by dhgarcia** [robotology/community#330] [Q&A] [answered]
We have a question about the operation of fingersTuner.

We try to run from yarpmanager the modules for iCubStartup  (yarprobotinterface, iKinCartesianSolver, iKinGazeCtrl, wholeBodyDynamics, gravityCompensator, fingersTuner, imuFilter)

fingersTuner always fail.
from terminal it fails with the following message:
```sh
[ERROR]group [left_hand] is missing!
[INFO]RFModule failed to open.
```

The hand does work with yarprobotinterface and yarpmotorcontrolgui,
How can we debug the issue?

Thanks and best regards

**Comment by pattacini** [comment, marked as answer]
Hi @dhgarcia 

Most likely, `fingersTuner` starts up without loading the correct configuration file. Instead, the file used seems to lack the `[left_hand]` group, as reported.

In the `icub-main` repo you can find the [correct configuration][1].

To check which configuration file will be loaded when launched, you can do (see #42 for more details):
```sh
$ yarp resource --context fingersTuner --from config.ini
```

[1]: https://github.com/robotology/icub-main/blob/master/app/fingersTuner/conf/config.ini

**Comment by dhgarcia** [comment]
Thank you @pattacini 
The configuration file loaded was missing both hands.

We updated it with the one in the `icub-main` repo and everything works fine now.

Thanks
